src/ServerScriptService/InstanceComponents/Misc/Sprint.lua

```diff
@@ -32,7 +32,7 @@ function Sprint:Destroy()
 end
 
 function Sprint:IsOwner(player: Player?)
-	return (not not player) or player == self._owner
+	return player == nil or player == self._owner
```

Comment:
Why does this return true if no player is passed in?

Reply:
The `:IsOwner()` function is used to determine of a particular "caller" is an owner and I use `nil` (like network ownership) to represent the server. The server is always an owner, it should always be authorized to call things and such. It's related to remotes, since our remotes are basically regular methods, and remote callbacks.

PR #42 will resolve this a bit better, and I'll refactor this after probably.